packages/web-app-pdf-viewer/src/index.js

```diff
@@ -0,0 +1,34 @@
+import translations from '../l10n/translations'
+import App from './App.vue'
+
+const routes = [
+  {
+    name: 'pdf-viewer',
+    path: '/:contextRouteName/:filePath*',
+    component: App,
+    meta: {
+      auth: false,
+      patchCleanPath: true
+    }
+  }
+]
+
+const appInfo = {
+  name: 'PDF viewer',
+  id: 'pdf-viewer',
+  icon: 'eye',
+  extensions: [
+    {
+      extension: 'pdf',
+      newTab: true,
```

Comment:
IMO it's not needed anymore to open in a new tab whenever the top bar is visible (which is always the case now). Also, the `closeApp` helper offers a way back to the source file in the files app with just one line of code.

Reply:
I don't think we will agree with this. We also consider opening in a new tab to be more consistent with other apps. Can I make it configurable? (the last time I looked at it, I think it would require some changes to pass the config here, but this would be an interesting thing to do for other apps as well).